### PR TITLE
[DO-1499] Implement my parts designs

### PR DESCRIPTION
**Pull request by viannbreslerAccso**
Implementation of the my parts figma desgins

### CODE REVIEW

File: frontend/src/app/modules/shared/components/card-icon/card-icon.component.html
Comment (JoelDietz) [inline frontend/src/app/modules/shared/components/card-icon/card-icon.component.html:1]: The alt tag should maybe filled with the name of the icon

File: frontend/src/app/modules/shared/components/input/input.component.html
Comment (JoelDietz) [inline frontend/src/app/modules/shared/components/input/input.component.html:1]: The alt tag should maybe filled with the name of the icon

File: frontend/src/app/modules/shared/components/parts-table/parts-table.component.html
Comment (JoelDietz) [inline frontend/src/app/modules/shared/components/parts-table/parts-table.component.html:1]: Just a remark the deleted Filtering has to be added back in but in changed form. But that will be done in an separate task when it got its new form.

File: frontend/src/app/modules/shared/components/severity/severity.component.html
Comment (JoelDietz) [inline frontend/src/app/modules/shared/components/severity/severity.component.html:1]: The alt tag should maybe filled with the name of the icon

**Review by JoelDietz — CHANGES_REQUESTED** [review]
The following things need still to be changed:

1. Select 2 lifecycle stages boarder color should be 4d4d4d but is rgba(0, 0, 0, 0.87)
2. Checkbox unselected unhovered boarder color should be #808080 but is rgba(0,0,0,0.38)
3. priority high between select and ID should be color #808080 but is black
4. Search bar boarder seems a different color but could not find the color used but it should be #DFDFDF
5. Search bar placeholder text seems a different color but could not find the color used but it should be #808080
6. The pagination overlays with the my parts detail page and the backdrop.

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:1]: Done

File: frontend/src/app/modules/shared/components/card-icon/card-icon.component.html
Comment (viannbreslerAccso) [reply frontend/src/app/modules/shared/components/card-icon/card-icon.component.html:1]: Done

File: frontend/src/app/modules/shared/components/input/input.component.html
Comment (viannbreslerAccso) [reply frontend/src/app/modules/shared/components/input/input.component.html:1]: Done

File: frontend/src/app/modules/shared/components/severity/severity.component.html
Comment (viannbreslerAccso) [reply frontend/src/app/modules/shared/components/severity/severity.component.html:1]: Done

**Review by JoelDietz — APPROVED** [review]

**Pull request merged**